projects/swimlane/ngx-ui/src/lib/components/toolbar/toolbar.component.ts

```diff
@@ -83,7 +48,8 @@ export class ToolbarComponent {
   }
```

Comment:
I think you forgot to put a new line between the end curly and the constructor.

Reply:
woops, updated